src/components/ExpandableCard.tsx

```diff
@@ -107,111 +36,121 @@ const ExpandableCard: React.FC<IProps> = ({
-              <Translation id="more" />
-            </ButtonLink>
-          )}
-          {isVisible && (
-            <ButtonLink onClick={() => setIsVisible(false)}>
-              <Translation id="less" />
-            </ButtonLink>
-          )}
-        </ButtonContainer>
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
-      >
-        <motion.div
-          variants={showHide}
-          animate={isVisible ? "expanded" : "collapsed"}
-          initial={false}
-        >
-          <Text
-            variants={fadeInOut}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
+      <AccordionItem borderTopWidth="0" flex="1">
+        <h3 style={{ margin: 0 }}>
+          <AccordionButton
+            width="100%"
+            padding={`1.5rem 1.5rem ${isVisible ? "0" : "1.5rem"}`}
+            flex="1"
+            onClick={onClick}
+            _hover={{
+              backgroundColor: "ednBackground",
+            }}
           >
-            {children}
-          </Text>
-        </motion.div>
-      </motion.div>
-    </Card>
+            <Box
+              display="flex"
+              width="100%"
+              alignItems="center"
+              flexDir={{ base: "column", sm: "row" }}
+            >
+              <Box
+                marginBottom={{ base: "0.5rem", sm: 0 }}
+                marginRight={{ base: 0, sm: "1rem" }}
+              >
+                <Box
+                  display="flex"
+                  alignItems="center"
+                  sx={{
+                    svg: { marginRight: "1.5rem" },
+                  }}
+                  margin="1rem 0"
+                >
+                  {!!Svg && <Svg />}
+                  <Text
+                    fontSize="xl"
+                    fontWeight="semibold"
+                    flex="1"
+                    marginTop="0rem"
+                    marginBottom="0.5rem"
+                  >
+                    {title}
+                  </Text>
+                </Box>
+                <Text
+                  fontSize="sm"
+                  color="text200"
+                  marginBottom="0"
+                  width="fit-content"
+                >
+                  {contentPreview}
+                </Text>
+              </Box>
+              <Text
+                fontSize="md"
+                color="primary"
+                ml={{ base: undefined, sm: "auto" }}
+                mt="auto"
+                mb="auto"
+              >
+                <Translation id={isVisible ? "less" : "more"} />
+              </Text>
+            </Box>
+          </AccordionButton>
+        </h3>
+        <AccordionPanel
+          paddingX="1.5rem"
+          paddingBottom="1.5rem"
+          paddingTop="0"
+          onClick={onClick}
+        >
+          <Fade in={isVisible}>
```

Comment:
Appreciate the effort to keep the same type of animation we had but I think that in this transition we can start using the default animation from the Chakra's Accordion component which makes this simpler and cleaner.
```suggestion
```